modules/phase_field/doc/content/modules/phase_field/CALPHAD.md

```diff
@@ -0,0 +1,36 @@
+# CALPHAD
```

Comment:
Is this page still valid?

Reply:
Yes it should be. @dschwen recently submitted a manuscript about the use of this capability in conjunction with a new sublattice KKS phase-field model